src/components/dev-hub/card-list.js

```diff
@@ -22,40 +24,99 @@ const ArticleCard = styled(Card)`
     flex: 1 1 360px;
 `;
 
+const VideoCard = styled(Card)`
+    flex: 1 1 360px;
+    cursor: pointer;
+`;
+
 const HasMoreButtonContainer = styled('div')`
     margin-bottom: ${size.large};
     margin-top: ${size.large};
     text-align: center;
 `;
 
-export default React.memo(({ items = [], limit = 9 }) => {
-    const [visibleCards, setVisibleCards] = useState(limit);
-    const hasMore = items.length > visibleCards;
-    return (
-        <>
-            <CardContainer>
-                {items.slice(0, visibleCards).map(item => (
-                    <ArticleCard
-                        to={item['slug']}
-                        key={item['_id']}
-                        image={withPrefix(item['atf-image'])}
-                        tags={getTagLinksFromMeta(item)}
-                        title={getNestedText(item['title'])}
-                        description={getNestedText(item['meta-description'])}
-                    />
-                ))}
-            </CardContainer>
-            {hasMore && (
-                <HasMoreButtonContainer>
-                    <Button
-                        secondary
-                        pagination
-                        onClick={() => setVisibleCards(visibleCards + limit)}
-                    >
-                        Load more
-                    </Button>
-                </HasMoreButtonContainer>
-            )}
-        </>
-    );
-});
+const getThumbnailUrl = media => {
+    return media.mediaType === 'twitch'
+        ? getTwitchThumbnail(media.thumbnailUrl, 1000)
+        : media.thumbnailUrl;
+};
+
+export default React.memo(
+    ({ videos = [], articles = [], podcasts = [], limit = 9 }) => {
+        const [visibleCards, setVisibleCards] = useState(limit);
+
+        //TODO: modify once the tab component is ready
+        const hasMore = videos.length
+            ? videos.length > visibleCards
+            : articles.length > visibleCards;
+
+        return (
+            <>
+                {articles.length > 0 && (
```

Comment:
Thanks for the changes to this component! Once we (later) combine these into one larger array this verbose code will be reduced for sure. Do you see where we are going with this component?

Reply:
Yes, I believe so. Perhaps we can talk more about this tomorrow to make sure we're on the same page

Reply:
Yep! We can go through the design and see how we can get the tab to work with the card list.